x-pack/plugins/endpoint/server/services/endpoint/alert_query_builders.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { KibanaRequest } from 'kibana/server';
+import { EndpointAppConstants } from '../../../common/types';
+import { EndpointAppContext } from '../../types';
+
+export const kibanaRequestToAlertListQuery = async (
+  pagingProperties: Record<string, any>,
+  endpointAppContext: EndpointAppContext
+): Promise<Record<string, any>> => {
+  const DEFAULT_TOTAL_HITS = 10000;
```

Comment:
Why `10,000`? Is it common to use this number as a default in ES?

Reply:
10000 is the default ES limit for simple pagination. It's also consistent with the defaults for the Endpoint Management APIs.

Reply:
@madirey is right